### DISCUSSION TITLE
`Unable to authenticate you` when attempting to run through hubs-cloud install on digital ocean.

### DISCUSSION THREAD

**Discussion by heaversm** [mozilla/hubs#3279] [Hubs Cloud Questions] [answered]
Posted [this issue](https://github.com/mozilla/hubs-cloud/issues/125) but in case this is a more appropriate location,

I'm unable to run through the setup script when I ssh into my droplet, receiving an `{ id: 'Unauthorized', message: 'Unable to authenticate you' }` message when I attempt to type in my writeable personal access token.

I've set up two domains for my hubs cloud instance and pointed them to my DO droplet DNS. They've also been added to networking in DO with an A record for the droplet IP and NS records. 

I created a personal access token in Digital Ocean with read and write permission with a name of [mytokenname] which is what I'm typing in at the prompt.

Perhaps related, if I attempt to run the setup script after failure (`.sh /opt/polycosm/setup.sh`) I get this error:

```
/opt/polycosm/setup.sh: 19: /opt/polycosm/setup.sh: [[: not found
internal/modules/cjs/loader.js:775
    throw err;
    ^

Error: Cannot find module '/root/dist/arbortect'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:772:15)
    at Function.Module._load (internal/modules/cjs/loader.js:677:27)
    at Function.Module.runMain (internal/modules/cjs/loader.js:999:10)
    at internal/main/run_main_module.js:17:11 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
/opt/polycosm/setup.sh: 26: /opt/polycosm/setup.sh: [[: not found
```

**Comment by robinkwilson** [comment, marked as answer]
This is very interesting, I definitely think the `{ id: 'Unauthorized', message: 'Unable to authenticate you' }` is the main issue you're seeing. 

Can you try tearing down and restarting the whole process?

**Reply by robinkwilson** [reply]
(Click **"Mark as unread"** when you reply! Thanks)
